.circleci/deployment/workflows.yml

```diff
@@ -37,21 +37,21 @@
           filters:
             branches:
               only:
-                - develop
+                - 1544-Nginx-with-domain # switch back to develop before merge
       - deploy-develop:
           requires:
             - deploy-infrastructure-develop
           filters:
             branches:
               only:
-                - develop
-      - test-deployment-e2e:
-          requires:
-            - deploy-develop
-          filters:
-            branches:
-              only:
-                - develop
+                - 1544-Nginx-with-domain # switch back to develop before merge
+      #- test-deployment-e2e:
+      #    requires:
+      #      - deploy-develop
+      #    filters:
+      #      branches:
+      #        only:
+      #         - 1544-Nginx-with-domain # switch back to develop before merge
```

Comment:
TODO: switch back to develop before merge

Reply:
reminder TODO